tests/step-032.sh

```diff
@@ -22,7 +22,7 @@ sed -i 's/^# Version: .*/# Version: 0/' /var/lib/githooks/base-template.sh &&
     exit 1
 
 OUTPUT=$(
-    sed -i 's|read -r EXECUTE_UPDATE </dev/tty|EXECUTE_UPDATE="N"|' /var/lib/githooks/base-template.sh &&
+    sed -i -E 's|show_prompt EXECUTE_UPDATE.*|EXECUTE_UPDATE="N"|' /var/lib/githooks/base-template.sh &&
```

Comment:
This could now be changed to 
```
OUTPUT=$(
    HOOK_NAME=post-commit HOOK_FOLDER=$(pwd)/.git/hooks ACCEPT_CHANGES=A EXECUTE_UPDATE=N \
        sh /var/lib/githooks/base-template.sh
)
```

Can you try it please? If it doesn't work on first try, just leave it, I'll sort it out later.